client/src/components/Slider.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+
+const Slider = ({
+  label,
+  levels,
+}: {
+  label: string;
+  levels: string[];
+}) => {
+  const [level, setLevel] = useState(0);
+
+  return (
+    <div className='flex flex-col gap-3'>
+      <label className='flex justify-between gap-3'>
```

Comment:
Accessibility issue as shown below:
<img width="475" alt="image" src="https://github.com/davidpak/SmartNote/assets/25410985/f2b9189c-b8d4-4392-9ec8-ecefe774e2e8">

You should either nest the `<input>` inside the `<label>` or associate the label with the input (I think you would need to set an `id` on the input and then `for` on the label